modules/afsocket/transport-mapper-unix.c

```diff
@@ -44,6 +46,18 @@ transport_mapper_unix_construct_log_transport(TransportMapper *s, gint fd)
     return log_transport_unix_stream_socket_new(fd);
 }
 
+static LogTransport *
+transport_mapper_unix_construct_log_transport(TransportMapper *s, gint fd)
+{
+  TransportMapperUnix *self = (TransportMapperUnix*) s;
+  LogTransport *transport = _construct_log_transport(s, fd);
+
+  if (self->pass_unix_credentials)
+    socket_set_pass_credentials(fd);
+
+  return transport;
+}
+
 static TransportMapperUnix *
```

Comment:
The _construct_log_transport should be renamed to _create_log_transport
and the transport_mapper_unix_construct_log_transport should be renamed to _construct_log_transport